examples/vjot.rb

```diff
@@ -29,18 +29,18 @@
         @note[0] = @title.text
         load_list
       end
-      stack :width => 1.0, :height => 200, :scroll => true do
```

Comment:
Hm. Using a float 1.0 to mean full-width is interesting here. Do we want to keep an unmodified copy of the legacy example, not just one modified to fit our current Scarpe?

Reply:
I'm not worried about the whitespace changes, of course. Those are fine.